### DISCUSSION TITLE
legal in germany ?

### DISCUSSION THREAD

**Discussion by Prokilo1987** [ReVanced/revanced-discussions#1059] [Question]
Is revanced legal in germany ? Is it banned there ? Can using it get you fined or in trouble ? And does using a vpn solve those problems in a certain way ?

**Comment by 09u2h4n** [comment]
Nope, i used vpn for Germany and everything works as smooth as butter.
Vpn works for you if there is a issue about region also it is breaking some rules of Google, what do you think why did Google close original Vanced app?

**Comment by Prokilo1987** [comment]
Are you sure? Wouldn't I get into legal trouble for using it in Germany?

Le dim. 1 sept. 2024 à 17:26, Oğuzhan Yılmaz ***@***.***> a
écrit :

**Reply by 09u2h4n** [reply]
Dude it is already illegal because of its purposes. There is no issues in europe tho.

**Comment by Prokilo1987** [comment]
So short answer is I can use it without risk or illegal consequences? U
sure bro? 🥺

Le lun. 2 sept. 2024 à 06:58, Oğuzhan Yılmaz ***@***.***> a
écrit :

**Reply by 09u2h4n** [reply]
How old are you bro? Short answer is if you want to use, use. If you don't want to use then don't use. And it seems you want to use then use it.

**Comment by oSumAtrIX** [comment]
ReVanced is not illegal. It is an open source patcher to modify apps of your choice. Legal matter depends on the app you patch. Patching YouTube violates the terms of service of YouTube, a contract you break which could lead to legal consequences.

**Comment by Prokilo1987** [comment]
What of I directly download youtube revanced app, is that a legal loophole?
Because it's technically a different app

**Reply by oSumAtrIX** [reply]
No/ even less legal, and there's no such thing as direct download from ReVanced. Unofficial sites are unsupported here. https://revanced.app

**Comment by Prokilo1987** [comment]
Sheeeesh bro 😢
So basically I'll have to rotate back to regular YouTube, Spotify, Netflix?
No way to keep my mod apps when I go to germany?

**Reply by oSumAtrIX** [reply]
Strictly legally speaking yes, effectively no

**Comment by Prokilo1987** [comment]
So if I just continue using them, will I be detected somehow? Or it's
illegal but everyone does it anyway without repercussions? Like if I just
continue using my mod apps in Germany, will they find out and send me a
warning or a fine? Can the government even find out about me using them
there?

**Comment by oSumAtrIX** [comment]
In theory yes, but effectively no. They wouldn't go that far, let alone ban
your account

**Comment by Prokilo1987** [comment]
So would they ban my account or wouldn't they if they find out I'm using
mod apps? Sry bro got confused plz be explicit and specific 😅

**Comment by oSumAtrIX** [comment]
In terms of service, yes, effectively no, like i said.

**Comment by Prokilo1987** [comment]
Bro plz stop sayong in terms of service and effectively, I don't know what
that means, would there be a ban or no? And would there be a fine or no?

**Comment by amyulating** [comment]
yeah buddy I used revanced and John YouTube came to my house and strangled me
